core/scripts/cli/sponsor/withdraw.js

```diff
@@ -47,7 +47,9 @@ const withdraw = async (web3, artifacts, emp) => {
     const collateralPerToken = toBN(collateral)
       .sub(toBN(withdrawRequestAmount))
       .mul(scalingFactor)
-      .div(toBN(position.tokensOutstanding.toString()));
+      .div(toBN(position.tokensOutstanding.toString()))
```

Comment:
Is this ceil'd or floor'd? Floor'd Im guessing, which we want because we should underestimate this fraction

Reply:
Floored. Which makes sense.